.gitignore

```diff
@@ -25,4 +25,5 @@ build/
 dist/
 
 data
+./public/bundle.js
```

Comment:
hey Anna, looks like the bundle was not git ignored as the file is added to this pr

I don't have the full context here but perhaps it should be `public/bundle.js` in git ignore

Reply:
not sure what's the problem, but I tried all the options and just submitted again and it's still there. was my personal preference but I think should be fine

Reply:
cool, I'm going to approve. If you want to remove from the repo, its a but of a pain, but you need to remove from the file system and commit the change to this branch. Up to you!

Reply:
thank you! yes, I tried to removing  it, adding to the gitignore and create again but it  still persists